### PR TITLE
Remove unnecessary search query parameter encoding

### LINKED ISSUE #10: Search query param encoding
# Bug Report

## My version of spotify-api.js
5.0.3

## What i expected...
Results for queries that contain spaces

## But what happened is...
```
const album = (await this.client.albums.search("Made In Lagos", {
      limit: 1,
})) as Album;
// Result: album = []
```
This search returned an empty response but when I made the equivalent curl request I received results. Tried a few more searches and noticed a pattern of queries with spaces not working.

### PR DESCRIPTION
**Pull request by shedrachokonofua**
Fixes #10 

Dug around and noticed axios internally encodes the values of the params object. Doing so before passing the values to Axios, as is currently done, is essentially encoding the value twice. Evidently, this query:
```
const album = (await this.client.albums.search("Made In Lagos", {
      limit: 1,
})) as Album;
```
will result in a call to this URL:
```
https://api.spotify.com/v1/search?query=Made%2520In%2520Lagos&type=album&market=US&offset=0&limit=20
```

Take notice of the query parameter, ```query=Made%2520In%2520Lagos```
If one were to decode that value they would get, `Made%20In%20Lagos` as opposed to the expected `Made in Lagos` i.e. it is being encoded twice.

### CODE REVIEW

**Comment by Scientific-Guy** [comment]
Thanks for the pr, we will be adding in this feature in v6!
Secondly kindly wait for @abh80 to review this pr

**Review by abh80 — APPROVED** [review]

**Comment by GithubBoat** [comment]
@shedrachokonofua, Everything looks good here, Great Job 
 I am ready to merge this PR on your behalf if you want so to happen please post a comment saying 
 > ready to merge

**Comment by Scientific-Guy** [comment]
Merged because the user did not responded for lot of time

**Pull request merged**
